clients/unshaded/src/main/java/tachyon/client/file/AbstractTachyonFileSystem.java

```diff
@@ -58,7 +58,7 @@
   protected FileSystemContext mContext;
 
   /**
-   * Constructor, currently {@link TachyonFileSystem} does not retain any state
+   * Constructor, currently {@link FileSystem} does not retain any state
```

Comment:
Remove this comment? It seems like it should be in `FileSystem`, but it isn't very useful anyway.